x-pack/plugins/ml/server/models/results_service/anomaly_charts.ts

```diff
@@ -1055,9 +1055,11 @@ export function anomalyChartsDataProvider(mlClient: MlClient, client: IScopedClu
             // differently because of how the source data is structured.
             // For rare chart values we are only interested wether a value is either `0` or not,
             // `0` acts like a flag in the chart whether to display the dot/marker.
-            // All other charts (single metric, population) are metric based and with
+            // For single metric chart, we need to pass null values to display data gaps.
+            // All other charts are metric based and with
```

Comment:
`All other charts are metric based` ? Should that be `all other charts are distribution based` ? Or have I misunderstood?

Reply:
That is a good point, updated in [c5fd024](https://github.com/elastic/kibana/pull/194119/commits/c5fd024457efa847085fb9f0cc3ae34a341651ba#diff-badfc09c0b1a3e5a0aa7b0cf01bd8af0d7198ca859707782616c6a10b272b33aR1059)